src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -20,22 +34,32 @@
 
 @Component
 public class NewAddressReportedService {
+  private static final Logger log = LoggerFactory.getLogger(NewAddressReportedService.class);
 
   private final CaseService caseService;
   private final EventLogger eventLogger;
+  private final PubSubTemplate pubSubTemplate;
 
   @Value("${censusconfig.collectionexerciseid}")
   private UUID censusCollectionExerciseId;
 
   @Value("${censusconfig.actionplanid}")
   private UUID censusActionPlanId;
 
-  @Value("${uprnconfig.dummyuprnprefix}")
+  @Value("${uprnconfig.dummyuprnxprefix}")
```

Comment:
Don't think this was meant to be changed

Reply:
yeap, noticed this when my local tests failed :)